### PR TITLE
[Task#67874] Create model

### PR DESCRIPTION
**Pull request by namnt0413**
## Related Tickets:

-   [#67874](https://edu-redmine.sun-asterisk.vn/issues/67874)

## WHAT this PR does?
- Tạo model tương ứng với các bảng trong database

## Notes Impacted Areas:

-   N/A.

## Attachments:
Screenshot 2023-09-12 152842

### CODE REVIEW

**Comment by vutuankiet4599** [comment]
ok

**Comment by tuanngoc1601** [comment]
ok

**Comment by namnt0413** [comment]
ready

File: app/Models/Film.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Film extends Model
+{
+    use HasFactory;
+
+    protected $table = 'films';
+
+    protected $guarded = ['id','created_at','updated_at'];
```
Comment (ghost) [inline app/Models/Film.php]: enter new line

**Comment by namnt0413** [comment]
ready

**Pull request merged**